object_manager.go

```diff
@@ -15,7 +15,7 @@ type IBObjectManager interface {
 	AllocateNetworkContainer(netview string, cidr string, isIPv6 bool, prefixLen uint, comment string, eas EA) (netContainer *NetworkContainer, err error)
 	CreateARecord(netView string, dnsView string, name string, cidr string, ipAddr string, ttl uint32, useTTL bool, comment string, ea EA) (*RecordA, error)
 	CreateAAAARecord(netView string, dnsView string, recordName string, cidr string, ipAddr string, useTtl bool, ttl uint32, comment string, eas EA) (*RecordAAAA, error)
-	CreateZoneAuth(fqdn string, ea EA) (*ZoneAuth, error)
+	CreateZoneAuth(fqdn string, nsGroup string, restartIfNeeded bool, comment string, SoaDefaultTtl int, SoaExpire int, SoaNegativeTtl int, SoaRefresh int, SoaRetry int, ea EA) (*ZoneAuth, error)
```

Comment:
Please, unify the format of the names. Some of them start from capital letter, some are not. I think, all of them should start from a small letter. Just for consistency.

Reply:
I've set all the field names to start with lower case